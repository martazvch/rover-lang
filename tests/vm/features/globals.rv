// Globals pure declaration
var name = "Adri"
var age = "24"

var lhs = 34.
var rhs = -.5

fn main() {
    print name + " is " + age // expect: "Adri is 24"
    print rhs * lhs // expect: -17

    // Change their value
    age = "27"
    name = "Martin"
    print name + " is " + age // expect: "Martin is 27"
}
