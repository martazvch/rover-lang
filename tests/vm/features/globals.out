Adri is 24
-17
Martin is 27
